package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@holdex/thirdweb-svelte",
-	"version": "0.0.11",
+	"version": "0.0.13",
```

Comment:
is it bug fixing? or breaking change? 
https://semver.org/

Reply:
for this feature, actually I think it makes sense for 0.1.0 @zolotokrylin
Ill change it to that then

Reply:
because its relatively big change